functional-test/src/test/groovy/org/rundeck/tests/functional/api/scm/ScmPluginActionsSpec.groovy

```diff
@@ -0,0 +1,174 @@
+        given:
+        String projectName = "${PROJECT_NAME}-P2"
+        setupProject(projectName, BASE_EXPORT_PROJECT_LOCATION)
+        GitScmApiClient scmClient = new GitScmApiClient(clientProvider).forIntegration(integration).forProject(projectName)
+        String jobUuid = getJobUuid(projectName, 'dummy-job')
+
+        expect:
+        scmClient.callSetupIntegration(GitExportSetupRequest.defaultRequest().forProject(projectName).withRepo(remoteRepo)).response?.success
+
+        when:
+        ScmActionInputFieldsResponse actionFields = scmClient.callGetFieldsForAction(actionId).response
+
+        then:
+        verifyAll {
+            actionFields.actionId == actionId
+            actionFields.integration == integration
+            actionFields.title == 'Commit Changes to Git'
+            actionFields.description == 'Commit changes to local git repo.'
+            !actionFields.importItems
+
+            actionFields.fields.size() == 3
+            actionFields.fields[0].toMap() == [
+                    defaultValue: null,
+                    description: "Enter a commit message. Committing to branch: `master`",
+                    name: "message",
+                    renderingOptions: [
+                            displayType: 'MULTI_LINE'
+                    ],
+                    required: true,
+                    scope: null,
+                    title: "Commit Message",
+                    type: "String",
+                    values: null
+            ]
+            actionFields.fields[1].toMap() == [
+                    defaultValue: null,
+                    description: "Enter a tag name to include, will be pushed with the branch.",
+                    name: "tagName",
+                    renderingOptions: [:],
+                    required: false,
+                    scope: null,
+                    title: "Tag",
+                    type: "String",
+                    values: null
+            ]
+            actionFields.fields[2].toMap() == [
+                    defaultValue: "true",
+                    description: "Check to push to the remote",
+                    name: "push",
+                    renderingOptions: [:],
+                    required: false,
+                    scope: null,
+                    title: "Push Remotely?",
+                    type: "Boolean",
+                    values: null
+            ]
+
+            actionFields.exportItems.size() == 1
+            actionFields.exportItems.first().toMap() == [
+                    deleted: false,
+                    itemId: "dummy-job-${jobUuid}.xml",
+                    job: [
+                            groupPath: null,
+                            jobId: "${jobUuid}",
+                            jobName: "dummy-job"
+                    ],
+                    status: "CREATE_NEEDED",
+                    originalId: null,
+                    renamed: false
+            ]
+        }
+
+        where:
```

Comment:
there is no point on having the integration/actionId in "where" block, since it's only 1 integration